src/test/java/org/gridsuite/study/server/service/client/AbstractWireMockRestClientTest.java

```diff
@@ -4,60 +4,49 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-
 package org.gridsuite.study.server.service.client;
 
 import com.github.tomakehurst.wiremock.WireMockServer;
+import lombok.AccessLevel;
+import lombok.Getter;
 import org.gridsuite.study.server.StudyApplication;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.AfterEach;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.cloud.stream.binder.test.TestChannelBinderConfiguration;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.ContextHierarchy;
-import org.springframework.test.context.junit4.SpringRunner;
 
 import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
 
 /**
  * @author Thang PHAM <quyet-thang.pham at rte-france.com>
  */
-@RunWith(SpringRunner.class)
 @SpringBootTest
 @ContextHierarchy({@ContextConfiguration(classes = {StudyApplication.class, TestChannelBinderConfiguration.class})})
 public abstract class AbstractWireMockRestClientTest {
-
-    public final Logger getLogger() {
-        return LoggerFactory.getLogger(this.getClass());
-    }
+    @Getter(AccessLevel.PROTECTED)
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
You broke the logging system, previously it was logging for the actual child class now it does not

Reply:
Strange I haven't see that. I'll revert that.